Directory.Build.props

```diff
@@ -1,4 +1,6 @@
 ﻿<Project>
+  <Import Project="eng\Common.props" />
```

Comment:
Without this, `$(BuildNative)` was only available in the projects that needed it when property was global ☹️